app/helpers/task_list_helper.rb

```diff
@@ -124,9 +124,12 @@ def row_helper(trainee, task)
       funding_active:
         {
           task_name: "Funding",
-          path: "#",
-          confirm_path: "#",
-          status: "not started",
+          path: edit_trainee_funding_training_initiative_path(trainee),
+          confirm_path: edit_trainee_funding_training_initiative_path(trainee),
+          status: ProgressService.call(
+            validator: Funding::TrainingInitiativesForm.new(trainee),
```

Comment:
Once the second funding page is in that @FriedSock is working on, we'll replace this with a higher level validation class, encompassing the logic re which sections of the funding flow need to be validated for the particular trainee (depending on bursary availability).